HPC/HPC.md

```diff
@@ -0,0 +1,24 @@
+# Getting started with HPC workloads on parallel clusters using Graviton CPUs
+
+**Introduction**
+
+HPC is a field where either high throughput and low latency is desired. Arm processor has its unique advantage in power efficiency and can find wide applications in HPC filed. Here is the instruction on how to get started with HPC on parallel clusters using Graviton CPUs.
```

Comment:
please stick to facts and not superfluous language. "This document describes how to get started running HPC applications on parallel clusters using Graviton EC2 instances." is sufficient.

Reply:
I have updated the description.